src/BootstrapAdminUi/templates/shared/helper/field/badge.html.twig

```diff
@@ -0,0 +1,26 @@
+{#
+    Badge rendering macro for grid fields.
+    
+    Accepts:
+    - Objects implementing BadgeableInterface (enums with getLabel(), getColor(), getIcon(), getValue())
+    - Arrays with keys: label, color, icon, value
+    - Simple strings
+    
+    Options can override labels, colors, and icons via options.vars.labels, options.vars.colors, options.vars.icons
+#}
+{% macro default(data, options) %}
+    {% set badgeData = sylius_badge_data(data, options|default({})) %}
+    
+    {% if badgeData %}
+        <span class="badge rounded-pill text-{{ badgeData.color }}" {{ sylius_test_html_attribute('badge', badgeData.value ?? badgeData.label) }}>
```

Comment:
I think the color should be on the badge itself (background instead of text), so for instance a colour  like `bg-purple-lt` would be rendered via 

```suggestion
        <span class="badge rounded-pill bg-{{ badgeData.color }}" {{ sylius_test_html_attribute('badge', badgeData.value ?? badgeData.label) }}>
```